### ISSUE TITLE
Local/Regional Volunteer Appointment Form/Workflow

### ISSUE THREAD

**Issue by jimgaffneyTT** [ThetaTau/CMT#774]
**tl;dr** We need a mechanism for RD's to nominate alumni to serve as local and regional volunteers.  This should workflow from the RD, to Ernestine to the admin assistant.

We're finalizing a proposal to bring back nationally-recognized local/regional volunteers (chapter advisors, assistant RD's, etc) like we did during the Bob Pope era.  

Under this proposal, RD's will be able to nominate alumni to serve in chapter-level and regional positions.  Those nominees will have to undergo the volunteer training program (which is what Jennifer is developing with an outside vendor) and the CommunityEdu/EverFi H&S training.  Once they do, they'll be vetted (to make sure they don't have a history of doing keg-stands on TV or something) and then have a quick convo with Ernestine.  Once they do those things, the CO (not the Council) will confirm their nomination and formally appoint them.

I think this would be a good use of workflows.  It would first go to Ernestine to administer the trainings, and once the trainings are complete, and after she does the vetting and the interview, she'll advance it to the admin assistant to write the appointment letters and such.
